outboundSender.go

```diff
@@ -142,6 +148,7 @@ type CaduceusOutboundSender struct {
 	maxWorkersGauge                  metrics.Gauge
 	currentWorkersGauge              metrics.Gauge
 	deliveryRetryMaxGauge            metrics.Gauge
+	querylatency                     metrics.Histogram
```

Comment:
Do we need this field?

Reply:
I don't think so. It was unused. Thanks!